.github/workflows/build-images-workflow-run.yml

```diff
@@ -259,6 +259,7 @@ jobs:
       PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
       GITHUB_REGISTRY_PUSH_IMAGE_TAG: ${{ github.event.workflow_run.id }}
       UPGRADE_TO_NEWER_DEPENDENCIES: ${{ needs.build-info.outputs.upgradeToNewerDependencies }}
+      CONTINUE_ON_PIP_CHECK_FAILURE: "true"
```

Comment:
We want to continue even if PIP_CHECK fails - so that we can get the image and continue testing regardless. We haave "Verify image" jobs that will show the failure in the CI.yml